lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -166,6 +170,7 @@ export abstract class ClientApplication {
 
         const responseHash = this.getRedirectResponseHash(hash || window.location.hash);
         if (!responseHash) {
+            this.logger.verbose("Hash returned not recognized or associated with redirect request, returning null");
```

Comment:
Maybe redundant with the log message 2 lines down?

Reply:
Yes! Must have missed it. Thanks for catching that. Removed